recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-bsp/u-boot/u-boot_%.bbappend

```diff
@@ -0,0 +1,23 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+LICENSE = "CLOSED"
+
+UBOOT_CONFIG[doma] = "xenguest_arm64_defconfig"
+UBOOT_CONFIG_prepend = "doma "
```

Comment:
What if we do not build DomA? Its u-boot is going to be built anyway

Reply:
fixed